evennia/accounts/accounts.py

```diff
@@ -753,7 +756,8 @@ def create(cls, *args, **kwargs):
         # Update the throttle to indicate a new account was created from this IP
         if ip and not guest:
             CREATION_THROTTLE.update(ip, 'Too many accounts being created.')
-        SIGNAL_ACCOUNT_POST_CREATE.send(sender=account, ip=ip)
+        SIGNAL_ACCOUNT_POST_CREATE.send(sender=account, ip=ip, creator=kwargs.get('creator', None),
```

Comment:
This is the wrong place to put this signal - because this method will call `utils.create.create_account` to create the account (where you also have a signal), this will lead to two signals being sent per creation.